### DISCUSSION TITLE
Effective Collaboration on Open Source Projects: Tips and Strategies

### DISCUSSION THREAD

**Discussion by rubyazz** [community/community#54418] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

What are some effective strategies for collaborating on open source projects with developers who have different skill sets or levels of experience, and how can you ensure that everyone is able to contribute meaningfully to the project?

**Comment by ingridinara** [comment]
Besides having a good documentation such as a well written and easy to understand README. CONTRIBUTTING & CODE OF CONDUCT, I would say that the crucial matter would be to stablish in the project goals with different levels of difficulties. By doing that, you open up an opportunity to less experienced devs to contribute as much as possible. By the way, you should not forget to add Tags such as helpneeded, so people could easily find it through GitHub s search bar.
